gammapy/maps/hpx.py

```diff
@@ -986,6 +986,22 @@ def to_binsz(self, binsz):
                 axes=copy.deepcopy(self.axes),
             )
 
+    def separation(self, center):
+        """Compute sky separation wrt a given center.
+
+        Parameters
+        ----------
+        center : `~astropy.coordinates.SkyCoord`
+            Center position
+
+        Returns
+        -------
+        separation : `~astropy.coordinates.Angle`
+            Separation angle array (2D)
```

Comment:
Minor: in the case of a `HPXGeom` the returned coordinate array should be 1D, maybe correct the docsting here.